scout/parse/panelapp.py

```diff
@@ -23,39 +22,14 @@ def parse_panel_app_gene(
     if confidence_level in PANELAPP_CONFIDENCE_EXCLUDE[confidence]:
         return gene_info
 
-    hgnc_symbol = panelapp_gene["gene_data"]["gene_symbol"]
-    ensembl_ids = [
-        version["ensembl_id"]
-        for genome in panelapp_gene["gene_data"]["ensembl_genes"].values()
-        for version in genome.values()
-    ]
-
-    if not ensembl_ids:  # This gene is probably tagged as ensembl_ids_known_missing on PanelApp
-        if hgnc_symbol in hgnc_symbol_ensembl_gene_map:
-            LOG.warning(
-                f"PanelApp gene {hgnc_symbol} does not contain Ensembl IDs. Using Ensembl IDs from internal gene collection instead."
-            )
-            ensembl_ids = [hgnc_symbol_ensembl_gene_map[hgnc_symbol]]
-        else:
-            LOG.warning(
-                f"PanelApp gene {hgnc_symbol} does not contain Ensembl IDs and gene symbol does not correspond to a gene in scout."
-            )
-
-    hgnc_ids = set(
-        ensembl_gene_hgnc_id_map.get(ensembl_id)
-        for ensembl_id in ensembl_ids
-        if ensembl_gene_hgnc_id_map.get(ensembl_id)
-    )
-    if not hgnc_ids:
-        LOG.warning("Gene %s does not exist in database. Skipping gene...", hgnc_symbol)
+    gene_symbol = panelapp_gene["gene_data"]["gene_symbol"]
+    hgnc_id = int(panelapp_gene["gene_data"]["hgnc_id"].split(":")[1])
+    if hgnc_id not in hgnc_gene_ids:
```

Comment:
Do you know the performance of this vs a dict? I would have thought it quite a lot faster than a long unordered list, potentially exhaustive search, but maybe the `in` operator is smart?

Reply:
Nah, linear time it seems - "container" dependent. So yes, let's use a `set`!